src/main/java/core/basesyntax/Animal.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public abstract class Animal {
+    public abstract void feed();
```

Comment:
let's add some state to our animal ( create fields that will store info about animal qualities : name, age ) 
and use inheritance for it

Reply:
done